docs/source/DEVELOPER.rst

```diff
@@ -144,3 +144,29 @@ The Deepcell Kiosk uses Google Kubernetes Engine to requisition resources on Goo
+    - A final variable in the benchamrking pod's YAML file is `COUNT`, which determines how many times the `zip100.zip` file will be submitted to the cluster for processing. This controls the total number of images the cluster will process over the course of the benchmarking run. For example, if you want to do a 10,000-image benchmarking run, you would set `COUNT` TO 100, since 100*100 = 10,000. Benchmarking data was presented in the paper for 10,000-image, 100,000-image, and 1,000,000-image runs. Accordingly, depending on which subset of our benchmarking data the user wishes to recreate, `COUNT` should be set to either 100, 1,000, or 10,000.
+
+2. Deploy a DeepCell Kiosk as you normally would. While navigating the cluster configuration menu, keep in mind that the option for Maximum Number of GPUs is relevant to the benchmarking process. In the paper, benchmarking data was presented for clusters with maxima of 1, 4, and 8 GPUs. Choose the appropriate maximum for the benchmarking dataset you would like to recreate.
+
+3. Once the cluster has finished deploying, drop to the shell via the Kiosk main menu and change the `values.replicas` field in `conf/helmfile.d/0410.benchmarking.yaml` from `0` to `1`. This tells the DeepCell Kiosk to deploy one benchmarking pod and, in the absence of any other input to the Kiosk (i.e., image submission via the command line, web interface, or ImageJ plugin), it will execute a full benchmarking run. Now redeploy the `benchmarking` deployment by executing `helmfile -l name=benchmarking sync`.
+
+5. Once `becnhmarking` has redeployed, the benchmarking process will begin. Benchmarking jobs can take a day or more, depending on the conditions chosen. To monitor the status of your benchmarking job, drop to the shell within the DeepCell Kiosk and execute the command `kubectl get logs benchmarking -- tail 100`. This will show you the most recent log output form the `benchmarking` pod. When benchmarking has finished, the final line in the log should be `Uploaded [FILEPATH] to [BUCKET] in [SECONDS] seconds.`, where `[FILEPATH]` is the location in `[BUCKET]`  where the benchmarking data has been saved.
+
+6. Now that data has been generated for your benchmarking run and saved in your bucket, you can analyze it. While we have a repository that generates the exact figures used in the paper (see the `version2` branch of the repository stored at github.com/vanvalenlab/publication-figures), that repository expects output files from all run conditions. Alternatively, if you just want to analyze a single run's data, you can download the output file from your bucket and look at two top-level fields in this large JSON file:
```

Comment:
This feels like way too much information, some of which could be moved to the figures repo. We should update master there so its not branch specific. I also am not a big fan of the second bullet point - we should fix the underestimate and point them directly to function - something that I think would best be done in the figure repo.

Reply:
I got a very weird error when trying to `git clone` the `publication-figures` repo. (Something about hte pull working but the checkout failing?) Otherwise, I would have made a PR for this merge. Can one of you confirm that pulling on that repo works for you?

Reply:
I just cloned the repository without an issue. I've also [submitted a PR](https://github.com/vanvalenlab/publication-figures/pull/1) for you to review and merge.